### ISSUE TITLE
Uncaught Error: The specified module could not be found./\?\~\AppData\Local\atom\app-1.51.0\res...

### ISSUE THREAD

**Issue by digique** [atom/atom#21285]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.51.0 x64
**Electron**: 5.0.13
**OS**: Microsoft Windows 10 Home
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: The specified module could not be found.
\\?\C:\Users\digiq\AppData\Local\atom\app-1.51.0\resources\app.asar.unpacked\node_modules\keyboard-layout\build\Release\keyboard-layout-manager.node

```
At electron/js2c/asar.js:138

Error: The specified module could not be found.
\\?\C:\Users\digiq\AppData\Local\atom\app-1.51.0\resources\app.asar.unpacked\node_modules\keyboard-layout\build\Release\keyboard-layout-manager.node
    at process.func [as dlopen] (electron/js2c/asar.js:138:31)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:881:18)
    at Object.func [as .node] (electron/js2c/asar.js:147:18)
    at Module.load (internal/modules/cjs/loader.js:701:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:633:12)
    at Function.Module._load (internal/modules/cjs/loader.js:625:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:14:16)
    at customRequire (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:1:722026)
    at get_KeyboardLayoutManager (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:14:2716581)
    at get_manager (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:14:2716683)
    at Object.getCurrentKeyboardLayout (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:14:2716809)
    at e.keystrokeForKeyboardEvent (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:14:1047862)
    at KeymapManager.keystrokeForKeyboardEvent (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:11:1228180)
    at KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:11:1226165)
    at WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.51.0/resources/app/static/<embedded>:11:284829)
```

### Commands

```
```

### Non-Core Packages

```

```

**Comment by sadick254** [comment]
@digique Are you still experiencing this issue?

**Comment by vincenttsungaa** [comment]
> @digique Are you still experiencing this issue?

I still do, I kept coming back in here just in case someone answers the issue

**Comment by Aerijo** [comment]
@vincenttsungaa Similar issues have found antivirus false positives to be the cause. Do you have antivirus active (and does it's logs mention anything about Atom if so)?

**Comment by vincenttsungaa** [comment]
> @vincenttsungaa Similar issues have found antivirus false positives to be the cause. Do you have antivirus active (and does it's logs mention anything about Atom if so)?

Yes it does active, but doesn't log anything about atom.

**Comment by digique** [comment]
And I only have essentials antivirus

**Comment by vincenttsungaa** [comment]
Solutions please. :<

**Comment by sadick254** [comment]
@vincenttsungaa The following have been known to work in the past. Maybe you could give them a try and let us know if the issue is resolved.
1. Reinstalling Atom.
2. If you have an antivirus installed on windows. You could check if `atom/Update.exe` is blocked by the antivirus and allow `atom/Update.exe` execution.
3. If on windows os. open the directory where your atom is located
I have this C:\Users_Your user name_\AppData\Local\atom
After that, go to the bin folder and open the atom.cmd file, just in case, click on apm.cmd.
Close atom and open it again, it should work.

**Comment by vincenttsungaa** [comment]
> @vincenttsungaa The following have been known to work in the past. Maybe you could give them a try and let us know if the issue is resolved.
> 
> 1. Reinstalling Atom.
> 2. If you have an antivirus installed on windows. You could check if `atom/Update.exe` is blocked by the antivirus and allow `atom/Update.exe` execution.
> 3. If on windows os. open the directory where your atom is located
>    I have this C:\Users_Your user name_\AppData\Local\atom
>    After that, go to the bin folder and open the atom.cmd file, just in case, click on apm.cmd.
>    Close atom and open it again, it should work.

Thank you so much sir! so much appreciated your help. The 3rd one worked!

**Comment by vincenttsungaa** [comment]
but the issue came back after a few minutes...

**Comment by retrohacker** [comment]
Seeing this on OSX as well:

[Enter steps to reproduce:]

1. Fresh install of OSx (have only added Bitwarden and Mulled VPN)
2. Download and install Atom
3. Went through GitHub token flow
4. Went to add GitHub project, immediately see a stream of these errors.

**Atom**: 1.58.0 x64
**Electron**: 9.4.4
**OS**: Mac OS X 10.14.6
**Thrown From**: Atom Core


### Stack Trace

Uncaught Error: dlopen(/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/keyboard-layout/build/Release/keyboard-layout-manager.node, 1): image not found

```
At electron/js2c/asar.js:140

Error: dlopen(/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app.asar.unpacked/node_modules/keyboard-layout/build/Release/keyboard-layout-manager.node, 1): image not found
    at process.func [as dlopen] (electron/js2c/asar.js:140:31)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1034:18)
    at Object.func [as .node] (electron/js2c/asar.js:149:18)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:74:18)
    at customRequire (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:1:756278)
    at get_KeyboardLayoutManager (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:14:2894476)
    at get_manager (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:14:2894578)
    at Object.getCurrentKeyboardLayout (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:14:2894704)
    at e.keystrokeForKeyboardEvent (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:14:1119649)
    at KeymapManager.keystrokeForKeyboardEvent (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:11:1224825)
    at KeymapManager.handleKeyboardEvent (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:11:1222810)
    at WindowEventHandler.handleDocumentKeyEvent (/private/var/folders/2m/s2vb3ltj11g4d8507hdg_k8w0000gn/T/AppTranslocation/869DC1C6-5892-4060-B8BE-B72709D1E1DE/d/Atom.app/Contents/Resources/app/static/<embedded>:11:286651)
```

### Commands

```
     -3:57.2.0 core:paste (input.input-text.native-key-bindings)
     -3:00.5.0 core:select-all (input.hidden-input)
     -1:39.9.0 github:toggle-git-tab (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
  5x -1:24.6.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```

```

**Comment by icecream17** [comment]
More duplicates: #22903 #22963 #23020 #23175 #23292 #23293 #23294 #23298 #23329 #23334

Also, this issue might be a duplicate of #14461. Wow
